CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-## 1.24.0-dev
+## 1.24.0
 * Feature - Configurable poll duration for container stats [@jcbowman](https://github.com/jcbowman) [#1646](https://github.com/aws/amazon-ecs-agent/pull/1646)
 * Feature - Add support to remove containers and images that are not part of ECS tasks [#1752](https://github.com/aws/amazon-ecs-agent/pull/1752)
+* Feature - Add ARM build script [#1747](https://github.com/aws/amazon-ecs-agent/pull/1747)
+* Feature - Add configurable prometheus endpoint for agent metrics [#1743](https://github.com/aws/amazon-ecs-agent/pull/1745)
```

Comment:
Why the number in bracket is #1743 while the link is for pr 1745?

Reply:
Good catch! will change the typo.